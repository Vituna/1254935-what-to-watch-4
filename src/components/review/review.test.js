import React from 'react';
import renderer from 'react-test-renderer';

import Review from './review.jsx';

const review = {
  text: `I didn&apos;t find it amusing, and while I can appreciate the creativity, it&apos;s an hour and 40 minutes I wish I could take back.`,
  author: `Amanda Greever`,
  date: `2015-11-18`,
  rating: `8,0`,
};

it(`Should Review render correctly`, () => {
  const tree = renderer
    .create(<Review
      review={review}
    />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
