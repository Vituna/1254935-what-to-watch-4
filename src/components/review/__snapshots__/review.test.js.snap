// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should Review render correctly 1`] = `
<div
  className="review"
>
  <blockquote
    className="review__quote"
  >
    <p
      className="review__text"
    >
      I didn&apos;t find it amusing, and while I can appreciate the creativity, it&apos;s an hour and 40 minutes I wish I could take back.
    </p>
    <footer
      className="review__details"
    >
      <cite
        className="review__author"
      >
        Amanda Greever
      </cite>
      <time
        className="review__date"
        dateTime="2015-11-18"
      >
        2015-11-18
      </time>
    </footer>
  </blockquote>
  <div
    className="review__rating"
  >
    8,0
  </div>
</div>
`;
